strutil.h

```diff
@@ -92,6 +92,32 @@ namespace strutil
         return result;
     }
 
+    /**
+     * @brief Converts std::string to capitalize case.

```

Comment:
Change `@brief` to `@brief Converts the first character of a string to uppercase letter and lowercases all other characters, if any.`